pandas/core/tools/datetimes.py

```diff
@@ -812,7 +804,9 @@ def to_datetime(
         When another datetime conversion error happens. For example when one
         of 'year', 'month', day' columns is missing in a :class:`DataFrame`, or
         when a Timezone-aware :class:`datetime.datetime` is found in an array-like
-        of mixed time offsets, and ``utc=False``.
+        of mixed time offsets, and ``utc=False``, or when parsing datetimes
+        with mixed time zones unless ``utc=True``. Parsing datetimes with mixed
+        time zones, please specify ``utc=True`` to opt in to the new behaviour.
```

Comment:
I think there's a word missing in this sentence? Like, "If parsing"? Also, no need to say "to opt in to the new behaviour" - this is the new behaviour :)

Reply:
Thanks for the comment. I corrected the wording as you suggested.